packages/sheet/src/placeholders.ts

```diff
@@ -0,0 +1,85 @@
+import { Tokens, UserTheme } from "./theme";
+
+const tokens: Tokens[] = [
```

Comment:
Considering future extensions to the `Tokens` type, it might be beneficial to define tokens with `as const` first, and then derive the `Tokens` type from it. This way, we can ensure type-level exhaustiveness checks automatically, making the code more robust against potential additions or modifications to the tokens array in the future.

```ts
export const tokens = [
  "colors",
  "fonts",
  "fontSizes",
  "fontWeights",
  "lineHeights",
  "letterSpacings",
  "spacings",
  "sizes",
  "radii",
  "zIndices",
  "breakpoints",
] as const;

export type Tokens = typeof tokens[number];
```